beets/library.py

```diff
@@ -259,7 +259,7 @@ class LibModel(dbcore.Model):
     """Shared concrete functionality for Items and Albums.
     """
 
-    _format_config_key = None
+    format_config_key = None
```

Comment:
I sort of liked the underscore prefix—it was consistent with the other "metadata" class variables for the Model classes like `_sorts`, `_types`, and `_fields`. I was trying to use this style to visually distinguish ordinary "data" fields from the metadata.